osu.Game/Skinning/SkinConfiguration.cs

```diff
@@ -16,5 +16,7 @@ public class SkinConfiguration : IHasComboColours, IHasCustomColours
         public Dictionary<string, Color4> CustomColours { get; set; } = new Dictionary<string, Color4>();
 
         public string HitCircleFont { get; set; } = "default";
+
+        public string CursorExpand { get; set; } = "1";
```

Comment:
Shouldn't this be a boolean instead of a string?

Reply:
I don't know how to get `CursorExpand` using `source.GetValue<SkinConfiguration, string>(s => s.CursorExpand)` with a boolean.

Reply:
`source.GetValue<SkinConfiguration, bool>(s => s.CursorExpand)`?

Reply:
I had this error : The type 'bool' must be a reference type in order to use it as parameter in the generic type or method.

Reply:
Oh, I found the problem : CursorExpand  has to be a nullable bool.